0.110.0/rockcraft.yaml

```diff
@@ -0,0 +1,54 @@
+# Based on: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol/Dockerfile
+name: opentelemetry-collector
+summary: Crucial component in the OpenTelemetry ecosystem.
+description: |
+  Designed to collect, process, and export telemetry data such as metrics, logs,
+  and traces from various sources to multiple destinations.
+version: "0.110.0"
+base: ubuntu@24.04
+license: Apache-2.0
+platforms:
+  amd64:
+services:
+  otelcol:
+    override: replace
+    summary: "Entry point for opentelemetry-collector oci-image"
+    startup: enabled
+    command: "/usr/bin/otelcol --config /etc/otelcol/config.yaml"
+    user: _daemon_
+run-user: _daemon_
+parts:
+  otelcol:
+    plugin: nil
+    source: https://github.com/open-telemetry/opentelemetry-collector-releases
+    source-type: git
+    source-tag: v0.110.0
+    build-snaps:
+      - go/1.23/stable
+    build-packages:
+      - ca-certificates
```

Comment:
So turns out we do need certs to build? Should we add a comment why it's needed?

Reply:
No, this is a leftover. Removed.